app/extensions/safe/ffi/lib.js

```diff
@@ -44,10 +44,6 @@ class LibLoader {
 
     return new Promise((resolve, reject) => {
       try {
-        if (os.platform() === 'win32') {
-          ffi.DynamicLibrary(path.resolve(__dirname, CONSTANTS.LIB_PATH.PTHREAD), mode);
```

Comment:
If we're not needing the lib, we should also remove the relevant lines from the `CONSTANTS.LIB_PATH`, too.